app/src/main/java/com/example/tasky/agenda/presentation/itemdetail/EventDetailScreen.kt

```diff
@@ -315,26 +315,32 @@ fun EventDetailScreen(
 }
 
 @Composable
-private fun AttendeesNotGoing(state: EventDetailState) {
+private fun AttendeesNotGoing(state: EventDetailState, onEvent: (EventDetailEvent) -> Unit) {
```

Comment:
Another option (instead of having 2 composables, attendeegoing and AttendeeNotGoing) you could have 2 AttendeeLists on your state

One thats the whole list
And one that gets filtered, when the user clicks on "All, Going or Not Going"
With this. You'd only have 1 composable, which gets updated by the 2nd list (the filtered one)